carbonmark/components/pages/Portfolio/Retire/RetireForm/index.tsx

```diff
@@ -94,25 +99,38 @@ export const RetireForm = (props: RetireFormProps) => {
 
     if (parts[0].toUpperCase() === "C3T") {
       return "c3";
+    } else if (parts[0].toUpperCase() === "ICR") {
+      return "icr";
     }
     return parts[0].toLowerCase();
   };
 
   const carbonTokenInfo =
     carbonmarkTokenInfoMap[getTokenPrefix(tokenSymbol) as CarbonmarkToken];
 
+  // temporary until digital-carbon subgraph credit its for ICR projects following registry-vintage format
```

Comment:
Can you create a ticket for this and link it here please?

Reply:
Combined ticket to keep track of these items; will break out post-merge: https://github.com/KlimaDAO/klimadao/issues/2093